src/web/mjs/connectors/MangaWT.mjs

```diff
@@ -11,4 +11,4 @@ export default class MangaWT extends MangaReaderCMS {
 
         this.language = 'tr';
     }
-}
\ No newline at end of file
+}
```

Comment:
Unfortunately the GitHub editor always appends a newline character at the end, therefore a LINT error occurs in the [automated check](https://github.com/manga-download/hakuneko/pull/1551/checks?check_run_id=600265662).
Try to remove the newline or change the LINT rules to allow newlines at the end of file.